pkg/log/log.go

```diff
@@ -0,0 +1,31 @@
+package log
+
+import (
+	"fmt"
+)
+
+type PrintLogger struct{}
+
+func (l PrintLogger) Actionf(m string, a ...interface{}) {
+	format(`►`, m, a...)
+}
+
+func (l PrintLogger) Waitingf(m string, a ...interface{}) {
+	format(`◎`, m, a...)
+}
+
+func (l PrintLogger) Successf(m string, a ...interface{}) {
+	format(`✔`, m, a...)
+}
+
+func (l PrintLogger) Warningf(m string, a ...interface{}) {
+	format(`⚠️`, m, a...)
+}
+
+func (l PrintLogger) Failuref(m string, a ...interface{}) {
+	format(`✗`, m, a...)
+}
+
+func format(tickmark, m string, a ...interface{}) {
+	fmt.Println(tickmark, fmt.Sprintf(m, a...))
+}
```

Comment:
do we need to have an instance of the logger? We could change it so they are just public functions instead, like we do in eksctl (using [kris-nova/logger](https://github.com/kris-nova/logger/blob/d961bfacb4a33574805af7cb715567442bdb58d7/logger.go#L109-L168))

Reply:
you're right, we don't! it was only needed when printing to stdout. thanks, refactoring now :)